aspnetcore/tutorials/min-web-api/samples/7.x/WebMinJson/Program.cs

```diff
@@ -0,0 +1,47 @@
+#define Second
+#if First
+#region snippet_1
+using Microsoft.AspNetCore.Http.Json;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Configure JSON options
+builder.Services.ConfigureHttpJsonOptions(options =>
+{
+    options.SerializerOptions.IncludeFields = true;
+});
+
+var app = builder.Build();
+
+app.MapGet("/", () => new Todo { Name = "Walk dog", IsComplete = false });
+
+app.Run();
+
+class Todo
+{
+    // These are public fields instead of properties.
+    public string? Name;
+    public bool IsComplete;
+}
+#endregion
+#else
+#region snippet_2
+using System.Text.Json;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+var options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
```

Comment:
@captainsafia According to our [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to?pivots=dotnet-6-0#case-insensitive-deserialization) documentation, ASP.NET Core apps automatically get web defaults, so the use of options explicitly set to web defaults actually has no effect. Should we keep it as-is or set other options to make it a more realistic example?

Reply:
Good catch! Let's try this so it's more realistic:

```suggestion
var options = new JsonSerializerOptions
{
  PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
  WriteIndented = true
}
```